packages/node/src/integrations/tracing/prisma.ts

```diff
@@ -119,6 +125,12 @@ export const prismaIntegration = defineIntegration(
         instrumentPrisma({ prismaInstrumentation });
       },
       setup(client) {
+        // If no tracing helper exists, we skip any work here
```

Comment:
this is not _perfect_ as possibly prisma could be set up later than `setup(client)` is being called, but it was the best I could come up with here 🤔 it seems to work in tests at least!